protocol/0024-OSTA-order_status.md

```diff
@@ -71,12 +80,12 @@ For a full outline of these behaviours, see [0037-OPEG-pegged_orders](./0037-OPE
 
 Note: The last row in the table above is added for clarity. If the order was filled, it is marked as Filled and it is removed from the book, so it can't expire after being filled.
 
-## Wash trading
-If, during continuous trading, an order would be filled or partially filled with an existing order from the same [party](./0017-PART-party.md) aka "wash" trade, the order is rejected. Any existing fills that happen before the wash trade is identified will be kept. FOK rules still apply for wash trading so if a wash trade is identified before the full amount of the order is complete, the order will be stopped and nothing filled.
-Wash trading is allowed on [auction](0026-AUCT-auctions.md) uncrossing. 
+## Wash trading ACs 
+- If, during continuous trading, an order would be filled or partially filled with an existing order from the same [party](./0017-PART-party.md) aka "wash" trade, the order is rejected. (<a name="0024-OSTA-005" href="#0024-OSTA-005">0024-OSTA-005</a>)
+- Any existing fills that happen before the wash trade is identified will be kept. (<a name="0024-OSTA-006" href="#0024-OSTA-006">0024-OSTA-006</a>)
```

Comment:
```suggestion
- Any existing fills that happen before the wash trade is identified will be kept. The order should be market both "partially filled" and "rejected to prevent wash trade" (<a name="0024-OSTA-006" href="#0024-OSTA-006">0024-OSTA-006</a>)
```